README.md

```diff
@@ -25,18 +24,32 @@ Maintained and developed with love by [André Gasser](https://fosstodon.org/@and
 
 # Why BigBone
 
-BigBone is a fork of [Mastodon4J](https://github.com/sys1yagi/mastodon4j), a Mastodon client library for Java and Kotlin that was published by Toshihiro Yagi. 
-The goal of Mastodon4J was to provide an easy-to-use library for interacting with the [Mastodon social media network](https://joinmastodon.org/) from Java and 
-Kotlin code. Unfortunately, it became abandoned and has not seen any updates since 2018. 
+BigBone is a fork of [Mastodon4J](https://github.com/sys1yagi/mastodon4j), a Mastodon client library for Java and Kotlin
+that was published by Toshihiro Yagi. The goal of Mastodon4J was to provide an easy-to-use library for interacting with 
+the [Mastodon social media network](https://joinmastodon.org/) from Java and Kotlin code. Unfortunately, it became 
+abandoned and has not seen any updates since 2018. 
+
+Since Elon Musk's Twitter acquisition in 2022, Mastodon has gained tremendous popularity. A project that is so well 
+received by the community deserves to have up-to-date and maintained client libraries. Because of this, we brought the 
+Mastodon4J project back to life in November 2022. That's when this project, the BigBone client library for Java and 
+Kotlin, was born. May it serve you well in your Mastodon-related endeavours!
+
+The name **BigBone** has mostly symbolic character. We have chosen the name BigBone for this library because Mastodons 
+represent impressive animals from the Pleistocene, built of big and heavy bones. At the same time, we hope this library 
+will build some sort of "skeleton" for your Mastodon-related projects. Interestingly, there is also 
+[Big Bone Lick State Park in Kentucky](https://parks.ky.gov/union/parks/historic/big-bone-lick-state-historic-site) 
+where American Mastodons have been excavated.
+
+# Project Team
+
+The project is currently taken care of by the following people:
 
-Since Elon Musk's Twitter acquisition in 2022, Mastodon has gained tremendous popularity. A project that is so well received by the community deserves to have 
-up-to-date and maintained client libraries. Because of this, we brought the Mastodon4J project back to life in November 2022. That's when this project,
-the BigBone client library for Java and Kotlin, was born. May it serve you well in your Mastodon-related endeavours!
+- [André Gasser](https://fosstodon.org/@andregasser)
+- [Andreas Bartels](https://fosstodon.org/@bocops)
+- [Patrick Geselbracht](@PattaFeuFeu@chaos.social)
```

Comment:
As suggested on Slack, I’d go for the GitHub profiles rather than our Mastodon accounts. Our role in this project is mainly as developers, even if we’re all also on Mastodon.

```suggestion
- [André Gasser](https://github.com/andregasser)
- [Andreas Bartels](https://github.com/bocops/)
- [Patrick Geselbracht](https://github.com/pattafeufeu/)
```